src/index.js

```diff
@@ -1,7 +1,13 @@
 window.onload = function() {
 	var ajax = new XMLHttpRequest();
 	ajax.open("GET", "banner.html");
-	ajax.responseType = "document";
 	ajax.send();
-	document.querySelector("#banner").innerHTML += ajax.responseXML;
-}
\ No newline at end of file
+	// When the ajax request changes state
+	ajax.onreadystatechange = function () {
+		// When it's now complete
+		if (ajax.readyState === 4) {
+			// Inject the response into the #banner element
+			document.querySelector("#banner").innerHTML = ajax.response;
```

Comment:
I changed your `+=` to an `=`. Since the point of the `#banner` component is only to contain the response from this request, it seems like it makes sense to do simple assignment instead of appending. Feel free to change this as you see fit, but assignment seems more correct to me.